src/parts/Media.tsx

```diff
@@ -31,12 +31,18 @@ export const FigCaption = ({video = false, playing = false, caption = ''}: {vide
   )
 }
 
-export const MediaImage = ({id, url, ratio, mediaClasses, className, alt = '', info = '', fit = false, style}: ImageProps) => {
+export const MediaImage = ({id, url, ratio, mediaClasses, className, alt = '', info = '', srcsetSize, fit = false, style}: ImageProps) => {
   const parsedInfos = info ? JSON.parse(info) : null
+  const srcSize = srcsetSize ? srcsetSize : null
+  const srcSet = parsedInfos?.srcset ? parsedInfos.srcset[srcSize] : null
+  const srcSizes = srcSet ? '(min-width: 768w) 768px, (min-width: 1024w) 1024px, (min-width: 1440w) 1440px' : ''
```

Comment:
Here again the srcset string is hardcoded… I thought we had these strings in the kirby config and were reusing them from there